.github/workflows/advanced.yml

```diff
@@ -105,6 +109,7 @@ env:
   BUILD_PATH: ${{ github.workspace }}/build
   CCACHE_DIR_PATH: ${{ github.workspace }}/ccache
   ARTIFACT_PATH: ${{ github.workspace }}/artifact
+  RELATIVE_ARTIFACT_PATH: ./artifact
```

Comment:
How is it different from ARTIFACT_PATH?

Reply:
The path to the cached directory is part of the (internal) cache key. If it's not relative, it leads to cache miss accross different runner OSes.